mslib/msui/mscolab.py

```diff
@@ -937,6 +937,11 @@ def delete_account(self, _=None):
         if reply == QMessageBox.No:
             return
         try:
+            auth_name = config_loader(dataset="MSCOLAB_auth_user_name")
+            del_password_from_keyring(self.mscolab_server_url, self.email)
```

Comment:
@matrss please have a look here. For cleanup that would be good to do. Do we have to add message box too, informing the user that we want to do that?

Reply:
I don't see a reason why an additional message box would be necessary. When the account is deleted the saved credentials no longer have any value, so why would someone want to retain them? IMO msui already has too many message boxes... Though the one asking for confirmation to actually delete the account makes sense.